configure.ac

```diff
@@ -74,8 +90,21 @@ AC_CHECK_LIB(dl, dlopen, [],
 dnl Checks for libraries
 AC_CHECK_LIB(pthread, pthread_mutex_init, [],
     AC_MSG_ERROR([pthread_mutex_init() not found.  libfabric requires libpthread.]))
-AC_CHECK_LIB(rt, clock_gettime, [],
-    AC_MSG_ERROR([clock_gettime() not found.  libfabric requires librt.]))
+
+have_clock_gettime=0
+have_host_get_clock_service=0
+
+AC_SEARCH_LIBS([clock_gettime],[rt],
+		[have_clock_gettime=1],
+		[AC_CHECK_FUNCS([host_get_clock_service],
+			[have_host_get_clock_service=1],
+			[AC_MSG_ERROR([clock_gettime or host_get_clock_service
+			 not found.])])])
+
+AC_DEFINE_UNQUOTED(HAVE_CLOCK_GETTIME, $have_clock_gettime,
```

Comment:
Minor quibble: put the `$have_clock_gettime` in m4 quoting (i.e., square brackets).  You don't *need* it here, but it's good to get in the habit of appeasing the m4 monster so that it won't bite you in the future.

```m4
AC_DEFINE_UNQUOTED(HAVE_CLOCK_GETTIME, [$have_clock_gettime],
                   [Define HAVE_CLOCK_GETTIME to 1 if available])
```